xil_carma_cloud/docker-compose.yml

```diff
@@ -0,0 +1,596 @@
+      - /opt/carma/vehicle/calibration:/opt/carma/vehicle/calibration
+      - /opt/carma/maps:/opt/carma/maps
+      - /opt/carma/routes:/opt/carma/routes
+      - /opt/carma/yolo:/opt/carma/yolo
+      - ./ns3_adapter/adapter_1/parameters.yaml:/opt/carma/install/ns3_adapter/share/ns3_adapter/config/parameters.yaml
+    command: >
+      bash -c "export RMW_IMPLEMENTATION=rmw_fastrtps_cpp
+      && wait-for-it.sh 172.3.0.2:11311 -- roslaunch ns3_adapter ns3_adapter.launch
+      role_id:='carma_1'
+      remap_ns:=/hardware_interface/comms"
+#############################################################################
+
+#############################################################################
+# carma_2 vehicle
+  roscore_2:
+    image: usdotfhwastol/carma-base:carma-system-4.5.0
+    networks:
+      carma_net_2:
+        ipv4_address: 172.5.0.2
+    container_name: roscore_2
+    volumes_from:
+      - container:carma-config:ro
+    volumes:
+      - /opt/carma/.ros/:/home/carma/.ros
+      - /opt/carma/logs/carma_2:/opt/carma/logs
+    restart: always
+    environment:
+      - ROS_IP=172.5.0.2
+      - ROS_MASTER_URI=http://172.5.0.2:11311/
+    command: roscore
+
+  platform_2:
+    image: usdotfhwastol/carma-platform:carma-system-4.5.0
+    networks:
+      carma_net_2:
+        ipv4_address: 172.5.0.3
+    container_name: platform_2
+    volumes_from:
+      - container:carma-config:ro
+    volumes:
+      - /opt/carma/logs/carma_2:/opt/carma/logs
+      - /opt/carma/.ros:/home/carma/.ros
+      - /opt/carma/vehicle/calibration:/opt/carma/vehicle/calibration
+      - /opt/carma/yolo:/opt/carma/yolo
+      - /opt/carma/maps:/opt/carma/maps
+      - /opt/carma/routes:/opt/carma/routes
+      - /opt/carma/data:/opt/carma/data
+      - /opt/carma/simulation:/opt/carma/simulation
+    environment:
+      - ROS_IP=172.5.0.3
+      - ROS_MASTER_URI=http://172.5.0.2:11311/
+    command: bash -c 'wait-for-it.sh 172.5.0.2:11311 -- roslaunch /opt/carma/vehicle/config/carma_docker.launch'
+
+  platform_ros2_2:
+    image: usdotfhwastol/carma-platform:carma-system-4.5.0
+    networks:
+      carma_net_2:
+        ipv4_address: 172.5.0.4
+    container_name: platform_ros2_2
+    volumes_from:
+      - container:carma-config:ro
+    volumes:
+      - /opt/carma/logs/carma_2:/opt/carma/logs
+      - /opt/carma/.ros:/home/carma/.ros
+      - /opt/carma/vehicle/calibration:/opt/carma/vehicle/calibration
+      - /opt/carma/maps:/opt/carma/maps
+      - /opt/carma/routes:/opt/carma/routes
+      - /opt/carma/yolo:/opt/carma/yolo
+      - ./carma/inlanecruising_config/parameters.yaml:/opt/carma/install_ros2/inlanecruising_plugin/share/inlanecruising_plugin/config/parameters.yaml
+      - ./carma/environment.launch.py:/opt/carma/install_ros2/carma/share/carma/launch/environment.launch.py
+    command: bash -c 'source /opt/carma/install_ros2/setup.bash && ros2 launch /opt/carma/vehicle/config/carma_docker.launch.py'
```

Comment:
its command looks different and missing middleware type export (vru config has it in multiple places, which we can copy, but really we should only need it for `platform_ros2_*` and `ros1_bridge_*` images, use:
```
    command: >
      bash -c 'export RMW_IMPLEMENTATION=rmw_fastrtps_cpp
      && source /opt/carma/install_ros2/setup.bash
      && ros2 launch /opt/carma/vehicle/config/carma_docker.launch.py'

Reply:
Done